opennms-services/src/main/java/org/opennms/netmgt/translator/EventTranslator.java

```diff
@@ -145,6 +145,7 @@ private void createMessageSelectorAndSubscribe() {
     /** {@inheritDoc} */
     @Override
     public void onEvent(Event e) {
+        final String eventString = LOG.isDebugEnabled() ? EventUtils.toString(e) : e.getUuid();
```

Comment:
Storing this in a variable seems a little misleading, it may be the event string or may be a uuid.